docs/platforms/react-native/manual-setup/expo.mdx

```diff
@@ -12,9 +12,17 @@ To set up the Sentry React Native SDK in your Expo project, follow the steps on
 - [Migrate from `sentry-expo` to `@sentry/react-native`](/platforms/react-native/migration/sentry-expo/).
 - [Sign up for an account](https://sentry.io/signup/).
 
-## Install the Sentry SDK
+## Automatic Install
 
-Install the `@sentry/react-native` package:
+[Sentry Wizard](https://github.com/getsentry/sentry-wizard) will patch your project automatically, though you can follow the [Manual Install](./#manual-install) if you prefer. You only need to patch the project once. Then you can add the patched files to your version control system.
+
+```bash {tabTitle:npm}
+npx @sentry/wizard@latest -i reactNative
+```
+
```

Comment:
does it make sense to add a `The following tasks will be performed by the Sentry Wizard` section similar to the [main getting started docs](https://docs.sentry.io/platforms/react-native/#install)?

Reply:
also maybe add a link to the #verify-setup section

Reply:
+1 to that.

Reply:
The link to verify setup is not necessary, the wizard prompts users to verify the setup and collect information about success/failure.

Reply:
The list of what wizard docs totally make sense, I'll add it, thank you.